composer.json

```diff
@@ -51,7 +51,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.8-dev"
+            "dev-master": "1.0-dev"
```

Comment:
1. keep one PR to change one thing.
2. version should be changed by maintainer, not by PRs.